src/main/java/ca/corefacility/bioinformatics/irida/service/export/ExportUploadService.java

```diff
@@ -300,12 +300,8 @@ public NcbiExportSubmission uploadSubmission(NcbiExportSubmission submission, St
 				for (SingleEndSequenceFile file : bsFile.getFiles()) {
 					// Just using file IDs as the basename for uploaded files to
 					// avoid accidentally sending sensitive sample names to NCBI
-					String filename;
-					if (file.getSequenceFile().getFile().toString().endsWith(".gz")) {
-						filename = file.getSequenceFile().getId() + ".fastq.gz";
-					} else {
-						filename = file.getSequenceFile().getId() + ".fastq";
-					}
+					String filename = file.getSequenceFile().getId() + ".fastq";
+
```

Comment:
Was this intended? It looks like it is undoing https://github.com/phac-nml/irida/pull/564

Reply:
Fixed thanks